common/common.go

```diff
@@ -27,9 +27,17 @@ var (
 	dataset      string
 	errorDataset string
 	filterFields []string
+	renameFields map[string]string
+	aliasFields  map[string]string
 	version      = "dev"
 )
 
+func ClearCache() {
```

Comment:
I think we could have a more descriptive name eg `ClearFieldMappingCache`. Also, as this is an exported type (capitalised first character), we should add a description.